definition_validator/validator.cc

```diff
@@ -189,20 +192,26 @@ optional<core::AutocorrectSuggestion> constructAllowIncompatibleAutocorrect(cons
 
     auto replaceLoc = ctx.locAt(parsedSig->sig.seen.override_);
 
+    if (!replaceLoc.exists()) {
+        return nullopt;
+    }
```

Comment:
not blocking: might be worth mentioning that this is the case if the child method is actually an `abstract` method, which is overriding a parent method